README.md

```diff
@@ -43,48 +47,34 @@ team to publish open source contributions and enhancements that are submitted to
 other libcrypto projects.
 
 ## AWS-LC features
+
 ### API Compatibility
+
 AWS-LC is compatible with the majority of OpenSSL’s APIs to make it easy to use with
 existing applications. We’re open to discussing adding missing functionality and
 understanding your use case in an Issue or on Gitter.
 
-### Compiler Compatibility
-AWS-LC’s libcrypto is a C library and is tested with a variety of C compilers. To
-build libssl and AWS-LC’s tests you will need a C++ compiler. AWS-LC is tested with:
+### Compiler, OS, and CPU support
 
-* GCC 4.8.5 to 8.4
-* Clang 7 to 10
-* Visual Studio 2015
-
-For a complete list of tested compilers see
+AWS-LC correctness is tested on a variety of C/C++ compiler, OS, and CPU
+combinations. For a complete list of tested combinations see
```

Comment:
Can you please modify the ci/Readme to add Android as a platform?  Have we added other platforms since?